routes/products.js

```diff
@@ -23,4 +41,55 @@ router.get('/:id', (req, res) => {
 		});
 });
 
+
+router.get('/', (req, res) => {
+	var productType = req.query.product;
+
+	Product.findOne({product: productType})
+		.then((doc) => {
+			if(!doc) {
+				const SPARQL = `
+        SELECT ?manufacturerLabel
+        WHERE
+        {
+          ?product ?label \"` + productType + `\"@en.
```

Comment:
Presumably you want to update this to parity with changes Mac has made on the database repository.

Reply:
See these:
https://github.com/reisman-internship-2019-greenmap/GreenMap-Database/blob/%238-esg-and-product-types/SPARQL%20Explicit%20Query%20Example.js

Reply:
Also this one:
https://github.com/reisman-internship-2019-greenmap/GreenMap-Database/blob/%238-esg-and-product-types/SPARQL%20Complicated%20Queries%20Example.js